crowdfunding_compassion/controllers/projects_controller.py

```diff
@@ -96,7 +142,8 @@ def project_creation(self, page=1, project_id=0, **kwargs):
            auth="public",
            type="http",
            method='POST',
-           website=True, noindex=['robots', 'meta', 'header'])
+           website=True, noindex=['robots', 'meta', 'header'],
```

Comment:
I think we can remove the noindex parameters and we will deinstall the problematic module.
```suggestion
           website=True,
```